cloud/organization/organization.go

```diff
@@ -0,0 +1,172 @@
+package organization
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	astro "github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/auth"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/astronomer/astro-cli/pkg/httputil"
+	"github.com/astronomer/astro-cli/pkg/input"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+)
+
+var (
+	httpClient = httputil.NewHTTPClient()
+
+	errInvalidOrganizationKey  = errors.New("invalid organization selection")
+	errInvalidOrganizationName = errors.New("invalid organization name")
+	AuthLogin                  = auth.Login
+)
+
+type OrgRes struct {
+	ID            string   `json:"id"`
+	Name          string   `json:"name"`
+	AuthServiceID string   `json:"authServiceId"`
+	Domains       []string `json:"domains"`
+	ShortName     string   `json:"shortName"`
+	CreatedAt     string   `json:"createdAt"`
+}
+
+func newTableOut() *printutil.Table {
+	return &printutil.Table{
+		Padding:        []int{44, 50},
+		DynamicPadding: true,
+		Header:         []string{"NAME", "ID"},
+		ColorRowCode:   [2]string{"\033[1;32m", "\033[0m"},
+	}
+}
+
+func listOrganizations(c *config.Context) ([]OrgRes, error) {
+	orgDomain := "https://api." + c.Domain + "/v1alpha1/organizations"
+	authToken := c.Token
+	ctx := context.Background()
+	doOptions := &httputil.DoOptions{
+		Context: ctx,
+		Headers: map[string]string{"authorization": authToken},
+	}
+	res, err := httpClient.Do("GET", orgDomain, doOptions)
+	if err != nil {
+		return []OrgRes{}, fmt.Errorf("could not retrieve organization list: %w", err)
+	}
+	var orgResponse []OrgRes
+	err = json.NewDecoder(res.Body).Decode(&orgResponse)
+	if err != nil {
+		return []OrgRes{}, fmt.Errorf("cannot decode organization list response: %w", err)
+	}
+
+	return orgResponse, err
+}
+
+// List all Organizations
+func List(out io.Writer) error {
+	c, err := config.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+
+	or, err := listOrganizations(&c)
+	if err != nil {
+		return errors.Wrap(err, astro.AstronomerConnectionErrMsg)
+	}
+
+	tab := newTableOut()
+	for i := range or {
+		name := or[i].Name
+		organizationID := or[i].ID
+
+		var color bool
+
+		if c.Organization == or[i].ID {
+			color = true
+		} else {
+			color = false
+		}
```

Comment:
```suggestion
		if c.Organization == or[i].ID {
			color = true
		}
```
`else` clause not needed as the default value of bool is `false`